src/zkevm_specs/evm/execution/sdiv_smod.py

```diff
@@ -0,0 +1,157 @@
+from ..instruction import Instruction, Transition
+from ..opcode import Opcode
+from ...util import FQ, RLC
+
+
+def sdiv_smod(instruction: Instruction):
+    opcode = instruction.opcode_lookup(True)
+
+    pop1 = instruction.stack_pop()
+    pop2 = instruction.stack_pop()
+    push = instruction.stack_push()
+
+    (
+        quotient,
+        divisor,
+        remainder,
+        dividend,
+        quotient_abs,
+        divisor_abs,
+        remainder_abs,
+        dividend_abs,
+    ) = gen_witness(opcode, pop1, pop2, push)
+    check_witness(
+        instruction,
+        quotient,
+        divisor,
+        remainder,
+        dividend,
+        quotient_abs,
+        divisor_abs,
+        remainder_abs,
+        dividend_abs,
+    )
+
+    instruction.step_state_transition_in_same_context(
+        opcode,
+        rw_counter=Transition.delta(3),
+        program_counter=Transition.delta(1),
+        stack_pointer=Transition.delta(1),
+    )
+
+
+def check_witness(
+    instruction: Instruction,
+    quotient: RLC,
+    divisor: RLC,
+    remainder: RLC,
+    dividend: RLC,
+    quotient_abs: RLC,
+    divisor_abs: RLC,
+    remainder_abs: RLC,
+    dividend_abs: RLC,
```

Comment:
these abs word should be derived from `instruction.abs_word`, eg.
```
    quotient_abs, quotient_is_neg = instruction.abs_word(quotient)
```

Reply:
Fixed to get the ABS values from `abs_word` function directly.